libs/platform/src/lib/components/action-bar/action-bar.component.ts

```diff
@@ -6,42 +6,21 @@ import {
     Output,
     EventEmitter,
     ChangeDetectorRef,
-    ViewChild,
-    ElementRef,
-    OnDestroy
 } from '@angular/core';
-import { Placement } from 'popper.js';
-
-export interface ActionItem {
-    label: string;
-    type: string;
-    priority: number;
-    editTitle: boolean;
-    options: string;
-    compact: boolean;
-}
-
+import { BaseComponent } from '../base';
 @Component({
-    host: {
-        '(document:click)': 'outSideClick($event)'
-    },
     selector: 'fdp-action-bar',
     templateUrl: './action-bar.component.html',
     styleUrls: ['./action-bar.component.scss'],
-    changeDetection: ChangeDetectionStrategy.OnPush
+    changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ActionBarComponent implements OnInit, OnDestroy {
+export class ActionBarComponent extends BaseComponent implements OnInit {
     /**
      * Actionbar title
      */
-    @Input()
-    title: string;
 
-    /**
-     * flag to set edit mode for renaming the title
-     */
     @Input()
-    editing: boolean = false;
```

Comment:
If we remove inputs then this is a breaking change => the PR should start with `feat:` and document all the breaking changes